north-america/README.NORTHAMERICA

```diff
@@ -47,8 +272,8 @@ grid coordinates are referenced to GR96.
 
 ### USA: NAD83 -> NAD83 (HARN/HPGN)
 
-*Source*: [.los/.las files coming from NOAA](https://www.ngs.noaa.gov/PC_PROD/NADCON/NADCON.zip)
-*Format*: NTv2
+*Source*: [.los/.las files coming from NOAA](https://www.ngs.noaa.gov/PC_PROD/NADCON/NADCON.zip)  
```

Comment:
There's a bunch of changes like this from here and below. Seems unnecessary to me but maybe there's a point?

Reply:
Yes, there are necessary for correct GitHub formatting. Otherwise the lines get concatenated

Reply:
Right, okay. Stuff like that usually catches my eyes instantly because I have my editor set up to remove trailing whitespace on save. So I would probably remove them by mistake if I were to edit that file...